### ISSUE TITLE
still Synchronizing ,never ends

### ISSUE THREAD

**Issue by Juanggv** [getumbrel/umbrel#1190]
I started my node in november 25 and still Synchronizing, some body can help me? I have a Raspberry Pi 4 connected by Ethernet, and a 1 Tera Solid Hard disk
the downloads data of blockchain data stops in 210.4 GB and never change

![image](https://user-images.githubusercontent.com/95034042/149014268-aa931c20-cce3-4f57-83ff-9457504c5960.png)

**Comment by wileyj** [comment]
I'm seeing this today after the latest upgrade. 
bitcoin container seems OK, will test rpc in a bit - i don't think this is a problem with the underlying services, but with the frontend as you've seen @Juanggv .

**Comment by jamesc2010** [comment]
I am having the same issue, got to 64% then just keeps cycling through the same 200 blocks, any assistance would be helpful

**Comment by wileyj** [comment]
2 things to look for: disk usage of the pi, and the logs of the bitcoin node. 


```
{9:16}~/Git/blockstack/public/stacks-blockchain:master ✓ ➭ ssh -A umbrel@umbrel
Linux umbrel 5.10.63-v8+ #1459 SMP PREEMPT Wed Oct 6 16:42:49 BST 2021 aarch64

umbrel@umbrel:~ $ df -h /mnt/data/
Filesystem      Size  Used Avail Use% Mounted on
/dev/sda1       916G  492G  378G  57% /mnt/data

umbrel@umbrel:~ $ docker logs -f bitcoin
<sample output >
2022-01-20T17:09:49Z [miner] AddToWallet c3a9bd3b8f7681c5e4f3635db258775b9dedabf35be9851b0435eac839739f52  update
2022-01-20T17:10:11Z [miner] AddToWallet 2f0965139ec482359c0a1e9fe1e2a12dd99e2f78529237c97b7794674f796b47  new
2022-01-20T17:10:31Z Socks5() connect to 2409:8a00:2544:11b0::c34:8333 failed: general failure
2022-01-20T17:11:03Z [miner] AddToWallet 7863896e9f7bb50e303ab3994653bf4e2e701fd1faad4d7f6ff7bb179fae64c0  new
2022-01-20T17:11:57Z [miner] AddToWallet 3c2cf6e91aae3c167ec11677d2b6cde9549e4090d6709933b848b6ceb7c35d51  new
2022-01-20T17:13:10Z [miner] AddToWallet 4322a58316ff3292fbd4bb7c526a2c1b493d6cea877ed46bf3b4ef5d3750184a  new
2022-01-20T17:17:37Z [miner] AddToWallet 61aa286bd48b82dbfdfa9db5164f5753f470867342a30dfc6f94469a7e479e90  new
2022-01-20T17:17:48Z UpdateTip: new best=0000000000000000000a9a0b9b705bd9177ed715b2caf1bf9861cfe6d634a3dd height=719657 version=0x20000004 log2_work=93.298821 tx=704284501 date='2022-01-20T17:17:28Z' progress=1.000000 cache=19.6MiB(93516txo)
2022-01-20T17:17:48Z [miner] AddToWallet 61aa286bd48b82dbfdfa9db5164f5753f470867342a30dfc6f94469a7e479e90  
2022-01-20T17:17:48Z [miner] Transaction 4322a58316ff3292fbd4bb7c526a2c1b493d6cea877ed46bf3b4ef5d3750184a (in block 0000000000000000000a9a0b9b705bd9177ed715b2caf1bf9861cfe6d634a3dd) conflicts with wallet transaction 2f0965139ec482359c0a1e9fe1e2a12dd99e2f78529237c97b7794674f796b47 (both spend c3a9bd3b8f7681c5e4f3635db258775b9dedabf35be9851b0435eac839739f52:3)
2022-01-20T17:17:48Z [miner] Transaction 4322a58316ff3292fbd4bb7c526a2c1b493d6cea877ed46bf3b4ef5d3750184a (in block 0000000000000000000a9a0b9b705bd9177ed715b2caf1bf9861cfe6d634a3dd) conflicts with wallet transaction 7863896e9f7bb50e303ab3994653bf4e2e701fd1faad4d7f6ff7bb179fae64c0 (both spend c3a9bd3b8f7681c5e4f3635db258775b9dedabf35be9851b0435eac839739f52:3)
2022-01-20T17:17:48Z [miner] Transaction 4322a58316ff3292fbd4bb7c526a2c1b493d6cea877ed46bf3b4ef5d3750184a (in block 0000000000000000000a9a0b9b705bd9177ed715b2caf1bf9861cfe6d634a3dd) conflicts with wallet transaction 3c2cf6e91aae3c167ec11677d2b6cde9549e4090d6709933b848b6ceb7c35d51 (both spend c3a9bd3b8f7681c5e4f3635db258775b9dedabf35be9851b0435eac839739f52:3)
2022-01-20T17:17:48Z [miner] Transaction 4322a58316ff3292fbd4bb7c526a2c1b493d6cea877ed46bf3b4ef5d3750184a (in block 0000000000000000000a9a0b9b705bd9177ed715b2caf1bf9861cfe6d634a3dd) conflicts with wallet transaction 61aa286bd48b82dbfdfa9db5164f5753f470867342a30dfc6f94469a7e479e90 (both spend c3a9bd3b8f7681c5e4f3635db258775b9dedabf35be9851b0435eac839739f52:3)
2022-01-20T17:17:48Z [miner] AddToWallet 4322a58316ff3292fbd4bb7c526a2c1b493d6cea877ed46bf3b4ef5d3750184a  update
2022-01-20T17:18:11Z [miner] AddToWallet adb12a016d316d9decac4ad02e966151bd4acdf3449f8ab2af1a4a74e6b9fd8d  new
```

the only reason i can think of for the sync to have "paused" is that you're out of disk space, or there is an error in the bitcoin container.

**Comment by wileyj** [comment]
`docker ps` could also be helpful here - if the bitcoin container is restarting often, that could indicate some other problem (memory usage usually, syncing bitcoin takes more memory/cpu)

**Comment by jamesc2010** [comment]
only using 17% of storage space on a 2 TB SSD, Is there a way to just delete everything and start from the beginning? I'm new to raspberry Pi and was given a "Bitcoin Machine" for christmas so am still learning the process.

**Comment by wileyj** [comment]
yes, you can do that - you'd need to format the disk and re-flash the microsd to start from scratch (there may be an easier way, but i'm not aware of it. possibly in the umbrel dashboard?)

the most important thing to look at though is the bitcoin logs - if the node is not syncing, the logs will tell us *why* it's not syncing. resetting the data and syncing from genesis may resolve this, but you also may hit the same issue again. 

tl;dr - check the bitcoin logs and see what that says.

**Comment by nmfretz** [comment]
closing old support issue
